src/public/media.ts

```diff
@@ -407,4 +413,66 @@ export namespace media {
     const messageId = sendMessageRequestToParent('viewImages', params);
     GlobalVars.callbacks[messageId] = callback;
   }
+
+  /**
+   * Optional barcode configuration supplied to scanBarCode API to customize barcode scanning experience in mobile
+   * All properties in BarCodeConfig are optional and have default values in the platform
+   */
+  export interface BarCodeConfig {
+    /**
+     * Optional; Lets the developer specify the type of barcode to be scanned
+     * Default value is All i.e one dimensional and two dimensional(only QRcode) barcodes can be scanned
+     */
+    barCodeType?: BarCodeType;
+    /**
+     * Optional; Lets the developer specify the scan timeout interval in seconds
```

Comment:
* Optional; Is this a standard way of specifying optional params? If it will just appear as a comment in documentation better use it in complete sentence.

Reply:
yes this is the way optional params are specified in the repo. Will keep it as is.